cloud-functions/http_standardize_names/requirements.txt

```diff
@@ -30,6 +30,7 @@ oauthlib==3.2.0
 packaging==21.3
 pandas==1.4.3
 phdi @ git+https://github.com/CDCgov/phdi@ebdd43120eec138373bc36900cf6ea7e25aedc29
```

Comment:
I believe this can be changed to simply `phdi` now that it is available on pypi. Unless we need something on the `main` branch of `phdi` that is not yet available in the PyPi release I think we should start transitioning away from installing phdi from GitHub